source/features/profile-hotkey.tsx

```diff
@@ -14,6 +14,7 @@ void features.add(__filebasename, {
 	shortcuts: {
 		'g m': 'Go to Profile',
```

Comment:
And also we can change
```suggestion
		'g m': 'Go to my profile',
```